public/scripts/world-info.js

```diff
@@ -3534,6 +3535,12 @@ export async function getSortedEntries() {
         // Chat lore always goes first
         entries = [...chatLore.sort(sortFn), ...entries];
 
+        // Parse decorators
+        entries = entries.map((entry) => {
```

Comment:
I moved decorators parsing here, not to do it on every recursion step (plus decorators are lost after the first step).